src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ExplicitInitializationCheck.java

```diff
@@ -67,29 +93,25 @@
  * </p>
  * <pre>
  * public class Test {
- *   private int a = 0;
- *   private int b = 1;
- *   private int c = 2;
+ *   private int intField1 = 0; // ignored
```

Comment:
usually, an OK case is marked as such:
`// OK, those references are ignored`
(applies to entire example)

@strkkk please correct, if `ignored` is good here.

Reply:
@muhlba91

Sorry, apart form this suggestion do you know what is the reason of travis-ci failure?

Reply:
this issue is related to travis itself (as far as i can judge now). it can be they are hitting some rate limiting or a similar issue:
- https://travis-ci.community/t/continuous-maven-repo-403/3908/37
- https://github.com/travis-ci/travis-ci/issues/10053#issuecomment-579983072
- https://issues.sonatype.org/browse/MVNCENTRAL-4985

@rnveach @romani @strkkk - can you please rerun the failed jobs and clear the caches. it seems maven have whitelisted new travis ip addresses nearly a week ago (unless they got new addresses in the meantime again).

Reply:
@muhlba91 good point. Yes, something like `// OK, references for primitives are ignored

@ImmortalRabbit please correct